_templates/header.php

```diff
@@ -100,6 +100,7 @@
         <script>
             <?php include __DIR__.'/../scripts/jql.min.js'; ?>
             jQl.loadjQ('https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.0/jquery.min.js')
+            jQl.loadjQdep('https://cdnjs.cloudflare.com/ajax/libs/flipclock/0.7.8/flipclock.min.js')
```

Comment:
Once this is merged with master, we can update the loki branch to take it out so on release we are all clean.